cmd/databases.go

```diff
@@ -138,6 +140,108 @@ var (
 			return nil
 		},
 	}
+
+	databaseListUsers = cli.Command{
+		Name:     "database-list-users",
+		Category: "Addons",
+		Usage:    "Print database's users",
+		Flags:    []cli.Flag{&appFlag, &addonFlag},
+		Description: CommandDescription{
+			Description: "List the users of a database",
+			Examples: []string{
+				"scalingo --app myapp --addon addon-uuid database-list-users",
+			},
+		}.Render(),
+
+		Action: func(c *cli.Context) error {
+			currentApp := detect.CurrentApp(c)
+			utils.CheckForConsent(c.Context, currentApp, utils.ConsentTypeDBs)
+			addonName := addonUUIDFromFlags(c, currentApp, true)
+
+			err := dbUsers.List(c.Context, currentApp, addonName)
+			if err != nil {
+				errorQuit(c.Context, err)
+			}
+			return nil
+		},
+	}
+
+	databaseDeleteUser = cli.Command{
+		Name:      "database-delete-user",
+		Category:  "Addons",
+		ArgsUsage: "user",
+		Usage:     "Delete a database's user",
+		Flags:     []cli.Flag{&appFlag, &addonFlag},
+		Description: CommandDescription{
+			Description: "Delete the given user of a database",
+			Examples: []string{
+				"scalingo --app myapp --addon addon-uuid database-delete-user my_user",
+			},
+		}.Render(),
+
+		Action: func(c *cli.Context) error {
+			if c.Args().Len() != 1 {
```

Comment:
**issue:** in this case, an error should be returned. the exit code 0 means the operation was done successfully, which is not the case here

Reply:
wtf you're in fact right, but that is weird.. i really thought that there were a client that pointed this behaviour, and that we should handle it differently. this should return an exit code 1, but you are right : consistency is more important